### DISCUSSION TITLE
When will sync be available?

### DISCUSSION THREAD

**Discussion by u2m4c6** [realm/realm-kotlin#436] [General]
First of all, this is an amazing project and Realm + KMP is a great idea. Being able to use a database provider with shared code will make a big difference on cross-platform projects.

Obviously this repo is quite active and you all are working through lots of issues so this is not a complaint. But is there a rough timeline of when sync to MongoDB will be available? (In Beta, not production ready).

**Comment by cmelchior** [comment]
While we don't promise anything, Sync support is going to be one of our main focus areas for the next couple of months. I would guess we have preview API's ready within the next month and beta-level support in the next 2-3 months.

**Reply by u2m4c6** [reply]
That sounds great! I will probably start playing around with local storage in a side-project so I can help test out the preview API's when they are available.

Also, what is the plan in regards to Kotlin Native concurrency changes? Or maybe a better question is: do the proposed changes even affect Realm KMP?

**Reply by cmelchior** [reply]
The changes to the memory model don't look like they will have any effect on the public API as such. It will make some of our internal implementations easer as well as on the consuming side, so we are still looking forward to it.
